### PR TITLE
Distoptica

### PR DESCRIPTION
**Pull request by mrfitzpa**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mrfitzpa** [comment]
@conda-forge/help-python, ready for review!

File: recipes/distoptica/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/distoptica-{{ version }}.tar.gz
+  sha256: bbfbff33237f3e9fc07d3517c9784bb795179d4916d62ecfbcb01709ca707273
+
+build:
+  skip: True  # [win or py<38]
```
Comment (xhochy) [inline recipes/distoptica/meta.yaml:13]:
```suggestion
  noarch: python
```

**Review by xhochy — CHANGES_REQUESTED** [review]
This package should be noarch.

File: recipes/distoptica/meta.yaml (hunk 1 above)
Comment (mrfitzpa) [reply recipes/distoptica/meta.yaml:13]: I did that originally but the windows build wouldn't work. The problem has to do with the dependency `pytorch`. For Windows machines, conda-forge only has `pytorch-cpu` available so it fails to build upon searching for `pytorch`. You can see [here](https://anaconda.org/conda-forge/pytorch/) that there is no Windows build for pytorch. Should I proceed nonetheless?

Comment (ocefpaf) [reply recipes/distoptica/meta.yaml:13]: It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

**Comment by mrfitzpa** [comment]
@xhochy @ocefpaf Ready for review again --- and thanks for your help!

**Pull request merged**